renku/cli/dataset.py

```diff
@@ -65,31 +65,34 @@
 
     $ renku dataset add my-dataset git+ssh://host.io/namespace/project.git
 
-Sometimes you want to import just a specific path within the parent project.
-In this case, use the ``--target`` flag:
+Sometimes you want to import just specific paths within the parent project.
+In this case, use the ``--source`` or ``-s`` flag:
 
 .. code-block:: console
 
-    $ renku dataset add my-dataset --target relative-path/datafile \
+    $ renku dataset add my-dataset --source path/within/repo/to/datafile \
         git+ssh://host.io/namespace/project.git
 
-To trim part of the path from the parent directory, use the ``--relative-to``
-option. For example, the command above will result in a structure like
+The command above will result in a structure like
 
 .. code-block:: console
 
     data/
       my-dataset/
-        relative-path/
-          datafile
+        datafile
 
-Using instead
+You can use ``--destination`` or ``-d`` flag to change the name of the target
+file or directory. The semantics here are similar to the POSIX copy command:
+if the destination does not exist or if it is a file then the source will be
+renamed; if the destination exists and is a directory the source will be copied
+to it. You will get an error message if you try to move a directory to a file
+or copy multiple files into one.
```

Comment:
what about specifying a directory that doesn't exist as a destination? Is it created or you get an error?

Reply:
It is created like `mkdir -p`

Reply:
I tried with `--destination directory/` and if put the source into a file called `data/mydataset/directory` -- so I'm not sure how you can specify a directory as the destination?

Reply:
a directory that doesn't exist, I should say

Reply:
Well, actually if the destination is just one directory then a rename happens. If it is multi-level directory then all directories except the last one are created and then a rename to the last name happens:
`if the destination does not exist **OR** if it is a file then ...`.

It actually works like a copy command with the exception that in copy you cannot pass multi-level non-existing directories as destination bu here you can (and of course you don't have globs here).